qaframework-bdd-tests/src/test/resources/features/oclsubscriptionmodule/ocl-subscription-module.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Import Dictionary via subscription module
+
+  Scenario: The user should copy the subscription url from the Dictionary manager
+    Given the user goes to the Dictionary Manager App
```

Comment:
Do we have step definition class for these guys.

Reply:
https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/cypress/integration/cucumber/step_definitions/oclsubscriptionmodule/ocl-subscription-module.js